examples/github_actions_orchestration/README.md

```diff
@@ -0,0 +1,74 @@
+# 🏃 Run pipelines in GitHub Actions
+
+# 🖥 Run it locally
+
+## 👣 Step-by-Step
+### 📄 Prerequisites 
+
+In order to run this example, you need to install and initialize ZenML.
+
+```bash
+pip install zenml
+
+# Install ZenML integrations: choose one of s3/gcp/azure depending on where your artifact store is hosted
+zenml integration install github <s3/gcp/azure>
+
+cd <ROOT_OF_YOUR_GITHUB_REPOSITORY>
+
+
+# If your git repository does not contain an existing ZenML pipeline, we can use the sample pipeline from this example
+zenml example pull github_actions_orchestration
+cp zenml_examples/github_actions_orchestration/run.py .
+rm -rf zenml_examples
+git add run.py
+git commit -m "Add ZenML example pipeline"
+```
+
+### 🥞 Create a new GitHub Actions Stack
+
+```bash
+export GITHUB_USERNAME=<>
+export GITHUB_AUTHENTICATION_TOKEN=<>
+
+zenml orchestrator register github_orchestrator --flavor=github
+zenml container-registry register github_container_registry --flavor=github --uri=<CR_URI> --automatic_token_authentication=true
+
+zenml secrets_manager register github_secrets_manager --flavor=github --owner=<GITHUB_REPOSITORY_OWNER> --repository=<GITHUB_REPOSITORY_NAME>
+
+# Register a metadata store and a secret to connect to it
+zenml secret register mysql_secret --schema=mysql --user=<USERNAME> --password=<PASSWORD> --ssl_ca=<> --ssl_cert=<> --ssl_key=<>
```

Comment:
This can only be run once the secret manager is part of the active stack - this will have to be moved down to after the stack setting